pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -463,7 +463,12 @@ if [ "$action" = list-generations ]; then
              configurationRevision=""
         fi
 
-        build_date="$(date --date="@$(stat "$generation_dir" --format=%W)" "+%a %F %T")"
+        if [ -z "$json" ]; then
+            build_date="$(date --date="@$(stat "$generation_dir" --format=%W)" "+%a %F %T")"
+        else
+            # jq automatically quotes the output => don't try to quote it in output!
+            build_date="$(stat "$generation_dir" --format=%W | jq 'todate')"
```

Comment:
I just saw that the `flake.lock` file uses the seconds instead of a human and machine readable ISO 8601 format. My initial Idea was to have it sortable and machine, as well as human readable. but since the raw output isn't made for human readability I would also be fine with the seconds.

Reply:
I think using the dates from the flake would be also miss leading.